src/Core/Configurations/RuntimeConfigValidatorUtil.cs

```diff
@@ -0,0 +1,65 @@
+using System.Text.RegularExpressions;
```

Comment:
every new file needs license headers.
```
// Copyright (c) Microsoft Corporation.
// Licensed under the MIT License.
```

look how other files have this header.

Reply:
updated